### PR TITLE
[Merged by Bors] - Enable logging

### LINKED ISSUE #364: Enable Log Aggregation for Druid
See https://github.com/stackabletech/hbase-operator/issues/291 and https://github.com/stackabletech/docker-images/pull/283 for reference.

This is part of https://github.com/stackabletech/issues/issues/288

### PR DESCRIPTION
**Pull request by siegfriedweber**
# Description

Add log aggregation

Closes #364 

## Integration tests

[Build Status](https://ci.stackable.tech/view/02%20Operator%20Tests%20(custom)/job/druid-operator-it-custom/86/)

## Review Checklist

- [x] Code contains useful comments
- [x] CRD change approved (or not applicable)
- [x] (Integration-)Test cases added (or not applicable)
- [x] Documentation added (or not applicable)
- [x] Changelog updated (or not applicable)
- [x] Cargo.toml only contains references to git tags (not specific commits or branches)
- [x] Helm chart can be installed and deployed operator works (or not applicable)

Once the review is done, comment `bors r+` (or `bors merge`) to merge. [Further information](https://bors.tech/documentation/getting-started/#reviewing-pull-requests)

### CODE REVIEW

File: rust/crd/src/lib.rs [hunk 1]
```diff
@@ -47,19 +48,24 @@ pub const OPERATOR_NAME: &str = "druid.stackable.tech";
...
-pub const LOG4J2_CONFIG: &str = "log4j2.xml";
+pub const LOG4J2_CONFIG: &str = "log4j2.properties";
 
 // store directories
 pub const SYSTEM_TRUST_STORE: &str = "/etc/pki/java/cacerts";
 pub const SYSTEM_TRUST_STORE_PASSWORD: &str = "changeit";
 pub const STACKABLE_TRUST_STORE: &str = "/stackable/truststore.p12";
 pub const STACKABLE_TRUST_STORE_PASSWORD: &str = "changeit";
 pub const CERTS_DIR: &str = "/stackable/certificates";
+pub const LOG_DIR: &str = "/stackable/log";
```
Comment (maltesander) [inline rust/crd/src/lib.rs:65]: No strong opinion here but i think we always called it `STACKABLE_LOG_DIR`.

**Review by maltesander — COMMENTED** [review]
Minor things, will continue tomorrow!

File: rust/operator-binary/src/druid_controller.rs [hunk 2]
```diff
@@ -647,11 +682,17 @@ fn build_rolegroup_statefulset(
...
+        },
+    )?;
     // druid container builder
-    let mut cb_druid = ContainerBuilder::new(APP_NAME)
-        .context(FailedContainerBuilderCreationSnafu { name: APP_NAME })?;
+    let mut cb_druid = ContainerBuilder::new(&Container::Druid.to_string()).context(
+        FailedContainerBuilderCreationSnafu {
+            name: Container::Druid.to_string(),
+        },
+    )?;
```
[the remark this answers is not in the document]
Comment (siegfriedweber) [reply rust/operator-binary/src/druid_controller.rs]: Container names put into variables (d2059267535d018915dedc5c264a2b3d578ea9d4)

**Review by maltesander — APPROVED** [review]
LGTM!

**Comment by siegfriedweber** [comment]
bors merge

**Pull request closed without merging**